### DISCUSSION TITLE
Which Credentials for Authorization?

### DISCUSSION THREAD

**Discussion by BobC0728** [LeleDerGrasshalmi/FortniteEndpointsDocumentation#216] [Q&A] [answered]
First off: thank you for this information!

Question:
How do I correlate the AuthClients with the data I am looking for?
I am looking to start with: 
FortniteEndpointsDocumentation/EpicGames/FN-Discovery-Service/Creator

How do I know which AuthClient to use for that data or any other data?

Thank you.

**Comment by BobC0728** [comment]
I think I figured it out

**Reply by LeleDerGrasshalmi** [reply, marked as answer]
closing as resolved
however heres the anwser for others
> you can search for clients with a certain permission on https://egs.jaren.wtf
